From 4b8eaef9fae4e4bbeb8e6b539a08946a8dac6184 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:27:44 -0400
Subject: [tools] perf/tests: Compensate lower sample freq with longer test loop

Message-id: <1408462094-14194-296-git-send-email-jolsa@redhat.com>
Patchwork-id: 88297
O-Subject: [PATCH RHEL7.1 BZ1131394 295/325] perf tests: Compensate lower sample freq with longer test loop
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 3fe2130523b2e098085eb4d38cd5b737a97cbee6
Author: Adrian Hunter <adrian.hunter@intel.com>
Date: Tue Nov 12 11:45:21 2013 -0300

description
===========
Doesn't work for me:

./perf test -v 19
19: Test software clock events have valid period values    :
--- start ---
mmap size 528384B
mmap size 528384B
All (0) samples have period value of 1!
---- end ----
Test software clock events have valid period values: FAILED!

Compensate the lower freq introduced in 67c1e4a53b17 with a longer loop,
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/tests/sw-clock.c b/tools/perf/tests/sw-clock.c
index 93a7139..6664a7c 100644
--- a/tools/perf/tests/sw-clock.c
+++ b/tools/perf/tests/sw-clock.c
@@ -9,7 +9,7 @@
 #include "util/cpumap.h"
 #include "util/thread_map.h"
 
-#define NR_LOOPS  1000000
+#define NR_LOOPS  10000000
 
 /*
  * This test will open software clock events (cpu-clock, task-clock)
-- 
1.7.1